src/components/Form/Form.spec.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Form from './Form';
+
+describe('Form component', () => {
+	it('should be displayed', () => {
+		render(<Form />);
+		const form = screen.getByRole('form');
+		expect(form).toBeInTheDocument();
+	});
+	it('should be clickable', async () => {
```

Comment:
I would rephrase this as your submit-button is clickable and not the whole form 🤔 
```suggestion
	it('should have a clickable submit-button', async () => {
```